OpenRA.Platforms.Default/StaticQuadIndexBuffer.cs

```diff
@@ -0,0 +1,63 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace OpenRA.Platforms.Default
+{
+	sealed class StaticQuadIndexBuffer : ThreadAffine, IDisposable, IIndexBuffer
+	{
+		uint buffer;
+		bool disposed;
+
+		public StaticQuadIndexBuffer(int size)
+		{
+			if (size % 6 != 0)
+				throw new ArgumentException("Buffer size must be a multiple of 6.");
+
+			OpenGL.glGenBuffers(1, out buffer);
+			OpenGL.CheckGLError();
+			Bind();
+
+			var indices = new uint[size];
+			var cornerVertexMap = new uint[] { 0, 1, 2, 2, 3, 0 };
+			for (var i = 0; i < indices.Length; i++)
+				indices[i] = cornerVertexMap[i % 6] + (uint)(4 * (i / 6));
+
```

Comment:
can you explain this a bit?
why 6 vertices? i know it's the same as the past code.